README.md

```diff
@@ -23,25 +22,27 @@ We recommend that you install TREX into a "virtual environment", which can be do
 `trex-venv` is the name of the directory that will be created and which will contain the virtual environment.
 You can choose a different name.
 
-Test the installation by running `trex-venv/bin/trex --version`.
-You can also choose to run `source trex-venv/bin/activate`, which will make the command directly available, that is, you can then simply type `trex` to run it. (Examples below assume you have done this.)
+Activate the virtual environment by running
+`source trex-venv/bin/activate`.
+You need to repeat this step in every new shell in order to be able to run TREX.
+Finally, test the installation by running `trex --version`.
```

Comment:
making the command directly available so that we can just write `trex` is not necessary anymore?

Reply:
The virtual environment activation is the step that makes it directly available. I omitted the sentence "you can simply type trex ..." because it’s now implicitly clar from the next line that this works because we ask the user to run `trex --help`.